guide/fr/appendices/file-formats.md

```diff
@@ -93,4 +93,4 @@ For data that changes frequently, and where each pull is limited in size, it is
 
 Like web services, databases provide direct access to data dynamically. Base de donnéess have the advantage that they can allow users to put together just the extraction that they are interested in.
 
-There are some security concerns about allowing remote database extraction and database access is only useful if the structure of the database and the importance of individual tables and fields are well documented. Often, it is relatively simple and inexpensive to create web services that expose data from a database, which can be an easy way to address safety concerns.
\ No newline at end of file
+There are some security concerns about allowing remote database extraction and database access is only useful if the structure of the database and the importance of individual tables and fields are well documented. Often, it is relatively simple and inexpensive to create web services that expose data from a database, which can be an easy way to address safety concerns.
```

Comment:
i am a newbie with github and trabnslate little by little ;-) this i will translate next week !